plugin.json

```diff
@@ -21,6 +21,13 @@
     "settings_schema": {
         "header": "The GitHub plugin for Mattermost allows users to subscribe to notifications, stay up-to-date with reviews, see the status of pull requests at a glance, and other common GitHub actions - directly from Mattermost. \n \n Instructions for setup are [available here](https://www.mattermost.com/pl/default-github-plugin#configuration).",
         "settings": [
+            {
+                "key": "UsePreregisteredApplication",
+                "display_name": "Use Preregistered OAuth Application:",
+                "type": "bool",
+                "help_text": "When true, instructs plugin to use preregistered GitHub OAuth application - application registration steps can be skipped. Requires Chimera Proxy URL to be configured for the server. Cannot be used with GitHub enterprise.",
+                "default": false
```

Comment:
Is there a desire to enable this by default for cloud?

Reply:
We do not want to enable it by default for on-prem for sure, so I am not sure if/how could we do it for cloud only. It is not super important tho but could be useful.

Reply:
IMO, yes this should be enabled by default in Cloud so the Admin could simply add Github, then install the app after first logging in,right?